Util/BuildTools/Setup.sh

```diff
@@ -92,15 +92,30 @@ for PY_VERSION in ${PY_VERSION_LIST[@]} ; do
     BOOST_PACKAGE_BASENAME=boost_${BOOST_VERSION//./_}
 
     log "Retrieving boost."
+
+    start=$(date +%s)
```

Comment:
Note that there is a simpe way in bash to measure and print the execution time of a command. Just writing `time <command_to_be_timmed>`:
```bash
time wget "https://archives.boost.io/release/${BOOST_VERSION}/source/${BOOST_PACKAGE_BASENAME}.tar.gz" || true
```

This command will print the output of the timmed command followed by this output:
```
real    0m0.015s
user    0m0.000s
sys     0m0.031s
```

The output of time command can be also modified by arguments:
https://man7.org/linux/man-pages/man1/time.1.html